business-central-tests/business-central-tests-gui/src/main/java/org/kie/wb/selenium/model/KieSeleniumTest.java

```diff
@@ -43,4 +48,16 @@ private static KieWbDistribution kieWbDistribution() {
                 .fromWarNameString(prop)
                 .orElseThrow(() -> new IllegalStateException("Invalid app.name='" + prop + "' Expecting business-central or business-monitoring"));
     }
+
+    public static int getHomepageLoadingTimeoutSeconds(final int defaultTimeout) {
```

Comment:
Why there is another one "default"? At this point we already have two: in constant and configurable in properties. Do we need more "default" values?